BUILD.md

```diff
@@ -1,91 +1,91 @@
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \ $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+  
+### Install the Docker Engine
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+
+### Check that the package has been correctly installed
+sudo docker run hello-world
 
-[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/5uuRTc3X4MM/0.jpg)](https://www.youtube.com/watch?v=5uuRTc3X4MM)
+```
 
-### Package structure - Web Services
+#### Installing nvm, nodejs and yarn
 
-[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/DAML56rx5yQ/0.jpg)](https://www.youtube.com/watch?v=DAML56rx5yQ)
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+nvm install --lts
 
-### Package structure - Main and Factory Plugin class
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt update && sudo apt install yarn
+```
 
-[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/w0bmkpge2Dw/0.jpg)](https://www.youtube.com/watch?v=w0bmkpge2Dw)
+#### Installing JDK 11
 
-### Package structure - Test class
+```
+sudo apt-get update && sudo apt-get upgrade
+sudo apt-get install openjdk-11-jdk
+java -version
 
-[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/3XpBYhN-8qs/0.jpg)](https://www.youtube.com/watch?v=3XpBYhN-8qs)
+### If you're using bash, you may need to set this in your bashrc file
+export JAVA_HOME=<path_to_java_home>
+```
 
-## Fast Developer Flow / Code Iterations
+For Windows, you can follow the guide provided [here](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html#GUID-371F38CC-248F-49EC-BB9C-C37FC89E52A0). For MacOS, you can follow the instructions [here](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-macos.html#GUID-F575EB4A-70D3-4AB4-A20E-DBE95171AB5F).
 
-We put a lot of thought and effort into making sure that fast developer iterations can be
-achieved (please file a bug if you feel otherwise) while working **on** the framework.
+#### Installing VS Code 
 
-If you find yourself waiting too much for builds to finish, most of the time
-that can be helped by using the `npm run watch` script which can automatically
-recompile packages as you modify them (and only the packages that you have
-modified, not everything).
+You can download and install VS Code from [here](https://code.visualstudio.com/download). It is suggested to install the git-graph and Docker plugins. 
 
-It also supports re-running the OpenAPI generator when you update any
-`openapi.json` spec files that we use to describe our endpoints.
+#### Clone and compile the Cactus repository 🌵
 
-The `npm run watch` script in action:
+```
+git clone https://github.com/hyperledger/cactus
+yarn add
```

Comment:
We can specify the `npm run install-yarn` prior to`yarn add` (yarn-add isn't required)